packages/inference/src/snippets/python.ts

```diff
@@ -100,16 +98,58 @@ const snippetGenerator = (templateName: string, inputPreparationFn?: InputPrepar
 		providerModelId?: string,
 		opts?: Record<string, unknown>
 	): InferenceSnippet[] => {
+		/// Hacky: hard-code conversational templates here
+		if (
+			model.pipeline_tag &&
+			["text-generation", "image-text-to-text"].includes(model.pipeline_tag) &&
+			model.tags.includes("conversational")
+		) {
+			templateName = opts?.streaming ? "conversationalStream" : "conversational";
+			inputPreparationFn = prepareConversationalInput;
+		}
+
+		/// Prepare inputs + make request
+		const inputs = inputPreparationFn ? inputPreparationFn(model, opts) : { inputs: getModelInputSnippet(model) };
+		const request = makeRequestOptionsFromResolvedModel(
+			providerModelId ?? model.id,
+			{ accessToken: accessToken, model: providerModelId, provider: provider, ...inputs } as RequestArgs,
```

Comment:
If I understand correctly, `model` is stripped out of the input

```suggestion
			{ accessToken, provider, ...inputs } as RequestArgs,
```

Reply:
Good catch yes. Not needed to pass it since we pass `resolvedModel` separately now